testsuite/benchmark-workloads/packages-experimental/experimental_usecases/sources/existence.move

```diff
@@ -0,0 +1,111 @@
+module 0xABCD::existence {
+    use std::signer;
+    use std::vector;
+    use std::string::{String, utf8};
+    use aptos_std::table;
+    use aptos_std::table::{Table};
+    use aptos_framework::event;
+
+    const ENOT_AUTHORIZED: u64 = 1;
+
+    struct A has key, store, copy, drop {
+        a1: u64,
+        a2: String,
+    }
+
+    struct B has key, store, copy, drop {
+        va: vector<A>,
+        name: String,
+        super_a: A,
```

Comment:
If there is any non-obvious reasons for these fields let's comment, o.w. fine (some dummy structure)